Izmenene (#9)
savannstm/sotws-translation: translation/skills.txt

causes an abominable regeneration!<#>начинает ужасающую регенерацию!
gives an encouraging cry!<#>произносит воодушевляющий клич!
hurled his pickaxe!<#>швыряет свою кирку!
grinds your flesh into pulp!<#>рубит вашу плоть в фарш!
cuts you up!<#>режет вас!
unleashes a blast of flames!<#>испускает поток пламени!
punches, and retrieves his blade!<#>атакует и возвращает своё оружие!
brandishes his blade!<#>размахивает своим клинком!
stabs into your flesh!<#>протыкает вашу плоть!
readies his lighter.<#>готовит свою зажигалку.
tossed Joel back at you!<#>бросает Джоэла обратно в вас!
is stroking Joel...<#>сжимает Джоэла...
is using Joel as a shield!<#>использует Джоэла, как щит!
tears you apart!<#>разрывает вас!
unleashes a fiery fist!<#>высвобождает кулак ярости!
flays you like a chicken!<#>снимает с вас кожицу, как с помидора!
is generating a current!<#>генерирует ток!
bites down on your neck!<#>кусает вас за шею!
They're watching...<#>За вами наблюдают...
is staring ominously...<#>угрожающее наблюдает...
cackles maniacally!<#>маникально смеётся!
lashes out!<#>набрасывается!
rams you with his pipe!<#>ударяет вас своей трубой!
is struggling to lift his pipe...<#>пытается поднять трубу...

> swings his flail!<#>машет цепом!

Reviewer: ```suggestion
swings his flail!<#>машет цепью!
```

swings his tire!<#>машет покрышкой!

> stomps him out!<#>вышибает предмет!

Reviewer: Речь, всё таки, про человека

gets ready.<#>готовится.
tosses a boomerang!<#>бросает бумеранг!
performs a boomerang combo!<#>исполняет комбо с бумерангом!
jabs at your eyes!<#>бьёт вам по глазам!
stabs you with a thumb!<#>ударяет вас большим пальцем!
emits a strange aura...<#>испускает странную ауру...
knocks the stuffing out of you!<#>выбивает из вас всё дерьмо!
reels back and swings out!<#>откидывается и машет!
beats you into the dirt!<#>втаптывает вас в грязь!
spat out a volley of flames!<#>выплёвывает залп пламени!
is grinning at you.<#>ухмыляется вам.

> strokes you...<#>ударяет вас...

Reviewer: Это, всё таки, обнимает

screams.<#>кричит.
is crying deeply.<#>горько плачет.
is glaring at you...<#>глядит на вас...
coughed up blood!<#>кашляет кровью!
realized he's outnumbered!<#>понял, что он в меньшинстве!
flails!<#>замахивается!

> vibrates with rage!<#>дрожит от ярости!

Reviewer: Надо получше что-то придумать, чем дрожит

> vibrates with rage!<#>преисполняется яростью!

Reviewer: ```suggestion
vibrates with rage!<#>наполняется яростью!
```

The enemy party was suddenly healed!<#>Вражеская команда внезапно исцелилась!
The enemy party was suddenly buffed!<#>Вражеская команда внезапно усилилась!
An enemy suddenly became limber!<#>Враг внезапно стал гибче!

> An enemy suddenly became solid!<#>Враг внезапно стал укрепился!

Reviewer: "стал укрепился"

prepares for his next attack...<#>готовится к следующей атаке...
stabs with a poisonous stinger!<#>колит ядовитым жалом!
pinches your skin!<#>щипает вашу кожу!
swung a sparkler!<#>машет бенгальским огнём!

> lashes with a cord!<#>взмахивает верёвкой!

Reviewer: Это, всё-таки, кабель

plays a truly beautiful song.<#>играет воистину прекрасную песню.
is tuning his violin.<#>настраивает свою скрипку.
tore you to shreds!<#>разрывает вас в клочья!
stabbed at your eyes!<#>ударяет вас по глазам!
swooped for your heads!<#>проходится по вашей голове!
lets out a piercing shriek!<#>издаёт пронзительный крик!
is roosting.<#>гнездится.
comes in with a divebomb!<#>кидается с прогибом!
is receiving a mysterious signal...<#>получает странный сигнал...
Disarming Slap<#>Обезоруживающий удар
Takes the "edge" off a wily assailant.\#Disarms an armed opponent.<#>"Притупляет" острый напор коварного агрессора.\#Обезоруживает вооружённого противника.
slaps at your wrists!<#>бьёт по запястью!
is trying to intimidate you.<#>пытается вас запугать.
flung salt in your eyes!<#>бросает соль вам в глаза!
aims between your eyes.<#>целится вам в голову.
cocks the hammer...<#>взводит курок...

> has given up.<#>сдаётся.

Reviewer: Мне кажется, лучше оставить в прошедшем времени

softens you up!<#>смягчает вас!
grabs at your weapon!<#>хватает ваше оружие!
stole your bat!<#>крадёт вашу биту!
couldn't break your grip!<#>не может вырваться из вашей хватки!
throws bottles everywhere!<#>бросает повсюду бутылки!
copulates fiercely!<#>яростно совокупляется!
spews out a fireball!<#>извергает огненный шар!
performs a Tiger Suplex!<#>выполняет тигровый суплес!
performs a Mongolian Chop!<#>выполняет монгольский удар!
pokes at the vital points!<#>ударяет в жизненно важные точки!
looks to the Heavens...<#>смотрит в небеса...
stomps you out!<#>пинает вас!
kicks!<#>пинается!
does a Gator Clamp!<#>выполняет зажим аллигатора!
chops!<#>рубит!

> is shrouded in smoke!<#>окутывается дымом!

Reviewer: Может, оставить "окутан"?

swings his cinderblock!<#>размахивает своим шлакоблоком!
strikes into your soul!<#>поражает вашу душу!
charges you!<#>делает в вас рывок!
gives a stiff arm!<#>машет сильнее!
unloads into you!<#>решетит вас!
flashes a disingenuous smile...<#>неискренне улыбается...
backhands you!<#>бьёт слева!
gets a party started!<#>начинает вечеринку!
Woo!<#>Уу!

> is lost...<#>теряется...

Reviewer: Тут тоже в прошедшем времени

Wooooooo!!!<#>Ууууууу!!!

> is astounded by your stupidity.<#>ошеломляется вашей тупостью.

Reviewer: И тут в прошедшем

scorches the earth all around!<#>выжигает землю вокруг!
swings his blade!<#>размахивает клинком!
crushes your skull!<#>давит ваш череп!
gets ready to fire!<#>готовится стрелять!
brings down the chopper!<#>сбивает вертолет!
steels himself...<#>закаляет себя...
launches you skyward!<#>запускает вас в небо!
brought you back down!<#>возвращает вас!

> sprayed paint in your eyes!<#>брызгает краску вам в глаза!

Reviewer: "Брызгает краской"?

is observing...<#>наблюдает...
is barking orders!<#>выкрикивает приказы!
slaps some sense into them!<#>вбивает здравомыслие!
sprayed paint every which-way!<#>разбрызгивает краску во все стороны!
protects his comrades!<#>защищает своих товарищей!
lines up his barrel...<#>направляет ствол...
shot out a burst of fire!<#>выстреливает очередью!
Silken Wrist<#>Шёлковое запястье
smacks with his wrist!<#>ударяет запястьем!
Ascending Claw<#>Восходящий коготь
thrusts up!<#>ударяет ввысь!
Descending Claw<#>Нисходящий коготь
thrusts down!<#>ударяет вниз!
Side Kick<#>Удар ногой
performs a side kick!<#>делает сайд-кик!
Flying Axe Kick<#>Удар летящего топора

> (WWD) Works your leg like a guillotine.\#Has a low chance of bleeding the enemy.<#>(WWD) Работайте ногой, как гильотиной.\#Имеет низкий шанс вызвать кровотечение у врага.

Reviewer: ```suggestion
(WWD) Works your leg like a guillotine.\#Has a low chance of bleeding the enemy.<#>(WWD) Работайте ногой, как гильотиной.\#Имеет низкий шанс вызвать у врага кровотечение.
```

hacks through air!<#>прорубает сквозь воздух!
Goat Kick<#>Козлиный удар
(DDD) A no-nonsense, double back kick.\#May launch enemy into the air. Costs 70 HP.<#>(DDD) Серьёзный удар обеими ногами.\#Может подбросить врага в воздух. Стоит 70 ОЗ.
delivers a goat-like buck!<#>лягает, словно козёл!
The Minor Iris<#>Малый Ирис
(WAS) The first supreme technique of the Velvet\#Style. Pierces defenses, but puts you off balance.<#>(WAS) Первая продвинутая техника Вельветового\#стиля. Пробивает защиту, но вы теряете равновесие.
thrashes about like mad!<#>мечется как сумасшедший!
Receding Stance<#>Стойка отклонения

> A disengaging stance, that helps regain focus.\#Recovers some SP.<#>Стойка выхода из боя, помогающая восстановить\#концентрацию. Также восстанавливает немного ОН.

Reviewer: Почему нельзя оставить "способствующая восстановлению концентрации" или "способствующая концентрации"?

Author: "способствующая восстановлению концентрации" - ненужное нанизывание падежей. Зачем способствующая, когда в сорсе буквально "помогает восстановить"?

Reviewer: > "способствующая восстановлению концентрации" - ненужное нанизывание падежей. Зачем способствующая, когда в сорсе буквально "помогает восстановить"?

Ну в контексте стойки, "помогает восстановить" звучит как-то нелогично. Вот если кто-то помогает, например, это да

closes distance!<#>сокращает дистанцию!
(WAS) The first supreme technique of the Velvet\#Lotus, precise strikes shred through solid defence.<#>(WAS) Первая высшая техника Вельветового лотоса - точные удары,\#пробивающие высокую защиту.
Fruit Toss<#>Бросок фрукта
Hurl what used to be fruit at an unsuspecting foe.\#Very low chance to poison.<#>Бросьте нечто, ранее бывшее фруктом, в ничего не\#подозревающего врага. Очень низкий шанс отравить.
chucks some rotten fruit!<#>бросает тухлый фрукт!
Putrid Meditation<#>Вонючая медитация
Embrace the filth. Restores some SP.<#>Станьте едины с грязью. Восстанавливает немного ОН.
finds inner peace!<#>находит внутренний мир!
Shag<#>Трахать

> Jiggy Jig<#>Шпили-вили

Reviewer: Это божественно

A basic dance. Boosts your confidence (and\#your attack).<#>Простой танец. Повышает вашу уверенность (и\#атаку).
gets down with his bad self!<#>выпускает свою злую сущность!
Glass Breaker<#>Стеклобой
Put your back into this one. Deals good damage,\#but can backfire. Chance to bleed.<#>Придётся поднапрячь спину. Наносит высокий урон,\#но может прилететь и вам. Шанс вызвать кровотечение.
beats them over the head!<#>бьёт по голове!

> Pumped-Up Jig<#>Заводная джига

Reviewer: Это прекрасно

parties like it's 1999!<#>веселится, будто это 90-е!
Mucus Blast<#>Взрыв слизи
Send out the Baby Fishman to blast an enemy with\#sticky mucus. Gets them wet and inflicts nastiness.<#>Отправьте малыша-рыболюда выстрелить по врагу\#липкой слизью. Враг намокает и ему становится противно.
directs the Baby Fishman!<#>направляет малыша-рыболюда!
Shock Current<#>Ударный ток
Command the Baby Fishman to zap them! Hits\#all enemies, but fries the Baby for a few turns.<#>Прикажите малышу-рыболюду поджарить их! Наносит урон\#всем врагам, но малыш сам поджаривается на пару ходов.
unleashes the Baby Fishman!<#>обрушивает малыша-рыболюда!
Command the Baby Fishman to zap them! Hits\#all enemies, but fries the Baby for a few turns.<#>Прикажите малышу-рыболюду поджарить врага! Наносит урон\#всем врагам, но малыш сам поджаривается на пару ходов.
unleashes the Baby Fishman!<#>выпускает малыша-рыболюда!
cuts his finger.<#>порезал свой палец.
Golden Finger<#>Золотой палец
Let the Midas touch turn all it blasts, into gold.\#The beauty of such a move can blind an enemy.<#>Пусть касание Мидаса превратит всё, что попадёт под его луч,\#в золото. Красота такого движения может ослепить врага.

> fires the ultimate beam!<#>стреляет ультимативным лучём!

Reviewer: Да, обосрался

Trowel Ravage<#>Разрушительная лопатка
Show them you're not fucking around.<#>Покажите врагу, что вы хуйнёй не маетесь.
gives 'em a thwacking!<#>задаёт им трёпку!
Check the Time<#>Уточнение времени
Look at your watch and pretend to not be a part of\#the fight. Restores SP and removes Taunt.<#>Посмотрите на свои часы и сделайте вид, что не участвуете\#в бою. Восстанавливает ОН и удаляет провокацию.
glances at his watch!<#>глядит на часы!
Spout<#>Плевок
Use the Baby Fishman to cool yourself off. Heals\#burning, washes off gunk, and makes you wet.<#>Используйте малыша-рыболюда, чтобы охладиться. Убирает\#горение, смывает всякую дрянь и мочит вас.
deploys the Baby Fishman!<#>выпускает малыша-рыболюда!
Full Stretch<#>Полная растяжка

> works out all the knots!<#>расслабляет все зажимы!

Reviewer: Это описания действия растяжки (буквально растяжки), так что может, что-нибудь получше есть?

Shank<#>Пырок
A quick stab to show them you mean business.<#>Быстрый удар ножом, намеренный показать врагам серьёзность.
stabs a bitch!<#>бьёт суку ножом!
Leg Stretches<#>Растяжка ног
Stretch those calves. Recovers TP and relieves\#tension. If tension is removed, TP is fully restored.<#>Растяните эти икры. Восстанавливает ОТ и убирает напряжение.\#Если убирает напряжение, полностью восстанавливает ОТ.
gets in a good stretch!<#>хорошенько растягивается!
White Streak<#>Белая полоса

> A slap of paint that's sure to disorient. Sets foes\#off balance and can even make them cry.<#>Красочный мазок, который точно дизориентирует врагов.\#Заставляет их потерять равновесие и заплакать.

Reviewer: ```suggestion
A slap of paint that's sure to disorient. Sets foes\#off balance and can even make them cry.<#>Красочный мазок, который точно дизориентирует врагов.\#Выбивает их из равновесия и заставляет плакать.
```

> A slap of paint that's sure to disorient. Sets foes\#off balance and can even make them cry.<#>Красочный мазок, который точно дизориентирует врагов.\#Выбивает их из равновесия и заставляет плакать.

Reviewer: ```suggestion
A slap of paint that's sure to disorient. Sets foes\#off balance and can even make them cry.<#>Красочный мазок, который точно дезориентирует врагов.\#Выбивает их из равновесия и заставляет плакать.
```

> A slap of paint that's sure to disorient. Sets foes\#off balance and can even make them cry.<#>Красочный мазок, который точно дизориентирует врага.\#Заставляет его потерять равновесие и заплакать.

Reviewer: Мульти-таргет атака

War Paint<#>Боевая раскраска
Let your foes know that you will bathe in their blood!\#Empowers your team, boosting attack and speed.<#>Дайте своим врагам знать, что вы будете купаться в их\#крови! Усиливает вашу команду, повышая атаку и скорость.
bellows a warcry!<#>издаёт боевой клич!
Swat<#>Шлепок
A quick smack to the cheek.<#>Быстрый шлепок по щеке.
gives 'em a slap!<#>шлёпает по щеке!
Spew<#>Рвота
Cough up some gunk onto an enemy. Gets them\#wet and has a chance to make them feel nasty.<#>Плюньте дрянью на врага. Мочит противников\#и имеет шанс заставить их чувствовать себя мерзко.
spits up some weird stuff!<#>изрыгает нечто странное!
Aromatherapy<#>Ароматерапия
Fill the air with the aroma of exotic salts.\#Heals many ailments and raises special defense.<#>Наполните воздух ароматом экзотических солей.\#Лечит множество недугов и повышает специальную защиту.
sways his censer...<#>качает кадилом...
Ingest<#>Поглощение
Chomp down on some tasty salt. Restores HP\#and SP, but only if you have salt in your inventory.<#>Сожрите немного вкусной соли. Восстанавливает\#ОЗ и ОН, но только если у вас в инвентаре есть соль.
gets ready to eat.<#>готовится есть.
Salt Wall<#>Соляная стена

> Deploy your impenetrable hide to protect your team!\#Redirects all damage to the user.<#>Разверните свою непробиваемую шкуру, чтобы защитить\#команду! Перенаправляет весь урон тому, кто использует навык.

Reviewer: Вообще, это эксклюзивный скилл краба, так что, может так и написать?

adopts a Fluid stance!<#>встаёт в жидкостную стойку!
pulls a power play!<#>проводит силовую игру!
performs a throw!<#>делает бросок!

> is too old.<#>чувствует на себе старость.

Reviewer: Ну как-то не очень. Может, сделать "слишком стар для этого"?
